presto-server-rpm/src/test/java/io/prestosql/server/rpm/ServerIT.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.server.rpm;
+
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+import static java.sql.DriverManager.getConnection;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+public class ServerIT
```

Comment:
TestServer or TestRpm

Reply:
It has to be named this way unless we change the default naming pattern: https://maven.apache.org/surefire/maven-failsafe-plugin/examples/inclusion-exclusion.html

I dislike `IT` but it seems better to use the defaults. Open to suggestions, though.